human-forces-provider/src/HumanForcesProvider.cpp

```diff
@@ -95,24 +206,21 @@ bool HumanForcesProvider::configure(yarp::os::ResourceFinder &rf)
         return false;
     }
     
+    human::GenericFrameTransformer frameTransform_fp1(inputFrame_fp1,outputFrame_fp1);
+    frameTransform_fp1.setTransform(transform_fp1);
+    
     human::FTForceReader *forceReader1 = new human::FTForceReader(appliedLink_fp1,
-                                                                  expressedFrame_fp1,
+                                                                  inputFrame_fp1,
                                                                   *firstSensor);
+    forceReader1->setTransformer(&frameTransform_fp1);
```

Comment:
I am afraid that this will cause a segfault at runtime.
You are passing the address of a variable allocated on the stack.